include/kllvm/codegen/CreateTerm.h

```diff
@@ -36,7 +36,8 @@ class CreateTerm {
    an llvm value corresponding to the specified KORE RHS pattern and substitution in the
    specified definition, and returns the value itself. */
   llvm::Value *operator()(KOREPattern *pattern);
-  llvm::Value *createToken(SortCategory sort, std::string contents);
+  llvm::Value *createToken(ValueType sort, std::string contents);
+  llvm::Value *createFunctionCall(std::string name, ValueType returnCat, std::vector<llvm::Value *> &args, bool sret, bool load);
```

Comment:
Please document. Is this for calling K functions? Are the arguments all supposed to be K terms? (or is this a utility for making an LLVM call instruction?)